pkg/nodeinfo/attributes.go

```diff
@@ -35,6 +35,7 @@ const (
 	NodeLabelCPUArch       = "kubernetes.io/arch"
 	NodeLabelMlnxNIC       = "feature.node.kubernetes.io/pci-15b3.present"
 	NodeLabelNvGPU         = "nvidia.com/gpu.present"
+	NodeLabelWaitOFED      = "nvidia.com/mofed.wait"
```

Comment:
I suspect we are going to have additional network-operator related labels in the future.

maybe we should go with a naming scheme that partitions  the network operator related labels.

e.g :  `network.nvidia.com/operator.mofed.wait` ? or similar

WDYT?

Reply:
Good idea! I didn't thought about it but it seems reasonable